components/text.css

```diff
@@ -0,0 +1,34 @@
+.text-utilities {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.text-utilities h1 {
+  font-size: 2rem;
+}
+.text-size {
```

Comment:
This has all the same properties with same values as the text-utilities class above, you can club them